.taskcluster.yml

```diff
@@ -142,17 +44,96 @@ tasks:
-            description: rust-code-analysis release build
+            name: rust-code-analysis linux release build
+            description: rust-code-analysis linux release build
+            owner: cdenizet@mozilla.com
+            source: ${repository}/raw/${head_rev}/.taskcluster.yml
+
+      - $if: 'tasks_for == "github-pull-request"'
+        then:
+          taskId: {$eval: as_slugid("build_windows_release")}
+          created: {$fromNow: ''}
+          deadline: {$fromNow: '2 hour'}
+          provisionerId: proj-relman
+          workerType: win2012r2
+          payload:
+            maxRunTime: 3600
+            env:
+              PROJECT_NAME: rust-code-analysis
+            command:
+              - set VSWHERE=%programfiles(x86)%\Microsoft Visual Studio\Installer
+              - set VS_BINS=VC\Tools\MSVC\*\bin\Hostx64\x64
+              - set RUSTUP_INIT_PATH=%CD%\rustup-init
+              - set CARGO_PATH=%USERPROFILE%\.cargo\bin
+              - set PATH=%RUSTUP_INIT_PATH%;%CARGO_PATH%;%VSWHERE%;%PATH%
+              - set VSWHERE_CMD=vswhere -latest -products * -find %VS_BINS%
+              - FOR /F "tokens=*" %%o IN ('%VSWHERE_CMD%') do (SET LIB_PATH=%%o)
+              - set PATH=%LIB_PATH%;%PATH%
+              - rustup-init -yv --default-toolchain stable ^
+                                --default-host x86_64-pc-windows-msvc
+              - git clone --recursive --quiet ${repository}
+              - cd rust-code-analysis
+              - git -c advice.detachedHead=false checkout ${head_rev}
+              - cargo build --all --release
+              - move target\release\rust-code-analysis-cli.exe target
+            mounts:
+              - content:
+                  url: https://win.rustup.rs/
+                file: rustup-init\rustup-init.exe
+            artifacts:
+              - name: public/rust-code-analysis-cli.exe
+                expires: {$fromNow: '2 weeks'}
+                path: rust-code-analysis/target/rust-code-analysis-cli.exe
+                type: file
+          metadata:
+            name: rust-code-analysis windows release build
+            description: rust-code-analysis windows release build
+            owner: cdenizet@mozilla.com
+            source: ${repository}/raw/${head_rev}/.taskcluster.yml
+
+      - $if: 'tasks_for == "github-pull-request"'
+        then:
+          taskId: {$eval: as_slugid("strip_windows_binary")}
+          dependencies:
+              - {$eval: as_slugid("build_windows_release")}
+          created: {$fromNow: ''}
+          deadline: {$fromNow: '2 hour'}
+          provisionerId: proj-relman
+          workerType: ci
+          payload:
+            maxRunTime: 3600
+            image: "rust:buster"
+            env:
+              PROJECT_NAME: rust-code-analysis
+            command:
+                - WINDOWS_BINARY=rust-code-analysis-cli.exe
+                - taskboot
+                - retrieve-artifact
+                - "--output-path=/tmp"
```

Comment:
Nit: you can use `--output-path=.` or `--output-path=$PWD` so you don't have to `cd /tmp` later